test/release/examples/primers/timers.chpl

```diff
@@ -16,9 +16,9 @@ use Time;
 config const quiet: bool = false;
 
 //
-// A ``Timer`` can be used like a stopwatch to time portions of code.
+// A ``stopwatch`` can be used like a stopwatch to time portions of code.
```

Comment:
Same redundancy here. Maybe just `a stopwatch can be used to time portions of the code`?

Reply:
I adjusted both cases for redundancy.